stl.go

```diff
@@ -186,10 +194,15 @@ func ReadFromSTL(i io.Reader) (o *Subtitles, err error) {
 	// Update metadata
 	// TODO Add more STL fields to metadata
 	o.Metadata = &Metadata{
-		Framerate: g.framerate,
+		CreationDate:           &g.creationDate,
```

Comment:
I forgot it in my previous review, but could you prefix `CreationDate` and `RevisionDate` with `STL`? (`STLCreationDate` and `STLRevisionDate` since they're `stl` only for now)

Don't forget to order fields alphabetically

Reply:
Done.